[2024-10-02T17:26:16.236+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T17:26:16.263+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T17:26:16.283+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T17:26:16.286+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T17:26:16.503+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-05 00:00:00+00:00
[2024-10-02T17:26:16.511+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1557) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T17:26:16.512+0000] {standard_task_runner.py:63} INFO - Started process 1570 to run task
[2024-10-02T17:26:16.512+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-05T00:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp60gqwu1c']
[2024-10-02T17:26:16.515+0000] {standard_task_runner.py:91} INFO - Job 132: Subtask transformar
[2024-10-02T17:26:16.564+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T17:26:16.640+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-05T00:00:00+00:00'
[2024-10-02T17:26:16.641+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T17:26:16.766+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T17:26:16.767+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/etl_pipeline.py", line 17, in transformar
    df_agrupada = df.groupby('ContinentExp')[['cases', 'deaths']].sum().reset_index()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py", line 8869, in groupby
    return DataFrameGroupBy(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/groupby.py", line 1278, in __init__
    grouper, exclusions, obj = get_grouper(
                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/grouper.py", line 1009, in get_grouper
    raise KeyError(gpr)
KeyError: 'ContinentExp'
[2024-10-02T17:26:16.779+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-05T00:00:00+00:00, execution_date=20231205T000000, start_date=20241002T172616, end_date=20241002T172616
[2024-10-02T17:26:16.790+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 132 for task transformar ('ContinentExp'; 1570)
[2024-10-02T17:26:16.806+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-02T17:26:16.824+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T17:26:16.827+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T18:58:33.335+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:58:33.366+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T18:58:33.578+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T18:58:33.579+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:58:33.598+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-05 00:00:00+00:00
[2024-10-02T18:58:33.605+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5889) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:58:33.608+0000] {standard_task_runner.py:63} INFO - Started process 5898 to run task
[2024-10-02T18:58:33.606+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-05T00:00:00+00:00', '--job-id', '1177', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmprxksgr90']
[2024-10-02T18:58:33.610+0000] {standard_task_runner.py:91} INFO - Job 1177: Subtask transformar
[2024-10-02T18:58:33.675+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:58:33.780+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-05T00:00:00+00:00'
[2024-10-02T18:58:33.783+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:58:33.916+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:58:33.919+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:58:33.935+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-05T00:00:00+00:00, execution_date=20231205T000000, start_date=20241002T185833, end_date=20241002T185833
[2024-10-02T18:58:34.104+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:58:34.137+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:58:34.142+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:24:35.473+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:24:35.708+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T19:24:35.715+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2024-10-02T19:24:35.716+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:24:35.730+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-05 00:00:00+00:00
[2024-10-02T19:24:35.736+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9185) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:24:35.738+0000] {standard_task_runner.py:63} INFO - Started process 9201 to run task
[2024-10-02T19:24:35.738+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-05T00:00:00+00:00', '--job-id', '2164', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpcrzkky_t']
[2024-10-02T19:24:35.740+0000] {standard_task_runner.py:91} INFO - Job 2164: Subtask transformar
[2024-10-02T19:24:35.783+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-05T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:24:35.876+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-05T00:00:00+00:00'
[2024-10-02T19:24:35.877+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:24:36.006+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:24:36.007+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:24:36.014+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-05T00:00:00+00:00, execution_date=20231205T000000, start_date=20241002T192435, end_date=20241002T192436
[2024-10-02T19:24:36.072+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:24:36.091+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:24:36.094+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
